assets/tilesets/tiles.tres

```diff
@@ -1,7 +1,19 @@
-[gd_resource type="TileSet" load_steps=3 format=3 uid="uid://du4mlit2n1o7c"]
+[gd_resource type="TileSet" load_steps=7 format=3 uid="uid://du4mlit2n1o7c"]
 
 [ext_resource type="Texture2D" uid="uid://c45d6tubm84m2" path="res://assets/tilesets/map_sprites/overworld_map.png" id="1_t8qnn"]
 
+[sub_resource type="TileMapPattern" id="TileMapPattern_oioag"]
```

Comment:
What are these patterns? I'd like to know so that I can use them. Could you send a screenshot?

Reply:
You can select stuff in Godot and then drag it into patterns. I'll show you how to use them in class